micro/go.mod

```diff
@@ -0,0 +1,29 @@
+// Licensed to SkyAPM org under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. SkyAPM org licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// Package micro (sw_micro) is a plugin that can be used to trace Go-micro framework.
+
+module swmicro
```

Comment:
I think the name of the module should be `github.com/SkyAPM/go2sky-plugins/go-micro`, what do you think?

Reply:
I think so